content/SCALE/SCALETutorials/SystemSettings/ManageBootEnvironSCALE.md

```diff
@@ -89,9 +88,10 @@ Each boot environment entry contains this information:
 * **Space**: shows boot environment size.
 * **Keep**: indicates whether or not TrueNAS deletes this boot environment when a system update does not have enough space to proceed.
 
-To access more options for a boot environment, click  <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> to display the list of options:
+Click the icon options for a boot environment to manage that environment.
```

Comment:
Reviewed a 25.04 environment and the options icon method no longer exists. There's a row of icons with one icon per action.
```suggestion
Use the icons row to take different actions for a boot environment.
```